### ISSUE TITLE
Unable to view manga on Kiss Manga after downloading around 100 chapters

### ISSUE THREAD

**Issue by ruudschouten** [raulhaag/MiMangaNu#148]
I was downloading all available chapters of Seitokai Yakuindomo on KissManga when the downloading suddenly stopped. I checked with other manga, but as soon as I opened a chapter I was shown the Next Chapter Previous Chapter window.
Now this kept happening, I tried removed the db's and reinstalling MiMangaNu, until I went to the site itself via my browser and selected a chapter. I was shown with a ReCaptcha "Are you a robot".
As soon as I checked it and went through the procedures I was able to download manga again.
I'm running on 1.39 by the way.

**Comment by raulhaag** [comment]
so if you fill the captcha in the browser it let you continue downloading?

**Comment by Blapple** [comment]
Yes

**Comment by Johndeep** [comment]
I could have a hint, maybe not about the "after 100 chapters", but about the captchas.
It's probably because of the security mechanism of cloudflare, on which KissManga and some other servers are running.

That's the cause why Kissmanga didn't run shortly last week. It was fixed by using IP, but nonetheless, the security layer of cloudflare comes with some methods to block those users. So we don't know when this method will fail again.
One of them is that ReCaptcha.. still don't know how to pass that.

Here you can read about a python tool, which can bypass that, but unfortunately also fail at the reCaptcha, https://github.com/Anorov/cloudflare-scrape

So, long story short, the ReCaptcha is not our doing ^^""

**Comment by raulhaag** [comment]
i downloaded all Seitokai Yakuindomo, with the "new" download system (realy an modification of actual for better feel), i don't know if is for my bad internet speed but i have no problems, the new download system are not ready, so it can be ready soon SFMBE
